extensions/amp-analytics/0.1/visibility-manager.js

```diff
@@ -702,9 +701,20 @@ export class VisibilityManagerForEmbed extends VisibilityManager {
     return this.backgroundedAtStart_;
   }
 
-  /** @override */
+  /**
+   * Gets the layout box of the embedded document. Note that this may be
+   * smaller than the size allocated by the host. In that case, the document
+   * will be centered, and the unfilled space will not be reflected in this
+   * return value.
+   * embed.iframe is used to calculate the root layoutbox, since it is more
+   * important for the embedded document to know its own size, rather than
+   * the size of the host rectangle which it may or may not entirely fill.
+   * embed.host is used to calculate the root visibility, however, since
```

Comment:
the embedded document is always embedded. 
We use `embed.host` to calculate the root visibility because the visibility of the host element directly determines the embedded document's visibility.

Reply:
Updated to use your wording. Thanks for the review!